src/main/webapp/resources/css/structure.css

```diff
@@ -1,4 +1,13 @@
 /*--------- BODY --------- */
+html {
+  /* Sticky footer */
+  position:relative;
+  min-height:100%;
+}
+body {
+  /* /* Sticky footer Margin bottom by footer height */
```

Comment:
Did you mean to have 2 open comment `/*`s?

Reply:
No. Thank you.

Reply:
Fixed.